easy_my_coop_initial_import/wizards/validate_subscription_requests/validate_subscription_requests.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+from odoo import api, fields, models
+
+from odoo.addons.queue_job.job import job
+
+log = logging.getLogger(__name__)
+
+
+class ValidateSubscriptionRequest(models.TransientModel):
+    _inherit = "validate.subscription.request"
```

Comment:
Since ou completely bypass the original wizard, I think it's better to use a server action that calls `enqueue_sr_validation`. The server action would be reached from the "more" menu on the subscription requests.

As a side note, I also think validate.subscription.request is useless, we could do the same without the wizard (but that's not in your code).